.github/CODEOWNERS

```diff
@@ -534,6 +534,8 @@
 
 # User Data Takeout.
 /core/domain/takeout_domain.py @vojtechjelinek @seanlip
+/core/domain/takeout_domain_jobs_one_off.py @vojtechjelinek @seanlip
+/core/domain/takeout_domain_jobs_one_off_test.py @vojtechjelinek @seanlip
 /core/domain/takeout_service*.py @vojtechjelinek @seanlip
```

Comment:
Should we do `/core/domain/takeout_*.py @vojtechjelinek @seanlip` here?

Reply:
+1, this consolidation seems sensible to me

Reply:
Done